userguide/sound_classifier/how-it-works.md

```diff
@@ -0,0 +1,52 @@
+# How Does this Work?
+
+Training and making predctions, for a sound classifier model, is a three
+stage process. The first stage does signal preprocessing. In
+the second step we use a pretrained neural network to extract deep
+features. Finally in the last step, we use a custom neural network.
```

Comment:
Nit: Maybe add bullet points here to improve readability? I know the rest of this document goes into each of the stages separately, but adding bullet points here can help the reader ease into that.

Reply:
That's a good idea. I decided to use a numbered list rather than a bulleted.